### ISSUE TITLE
APIModel property with name starting with "is" is getting ignored

### ISSUE THREAD

**Issue by mlvn23** [swagger-api/swagger-core#874]
I'm using "com.wordnik" % "swagger-play2_2.11" % "1.3.12" in Play, and if I use a property like ```is{{ Whatever }}``` in my API model, all the annotations are ignored:

```
@(ApiModelProperty @field)(required = true, value="true if open to the public")
isFoo: Boolean
```

becomes 

```
required: [
"name",
"instruments"
],
:
foo: {
type: "boolean"
}
```

If I renamed the property from isFoo to just foo, then it would render correctly:

```
required: [
"name",
"shared",
"instruments"
],
:
shared: {
type: "boolean",
description: "true if shared to the public"
},
```

**Comment by fehguy** [comment]
I believe this is fixed in develop_2.0.  Will verify

**Comment by fehguy** [comment]
This has been fixed in 1.5.1-M1 and tested in the (new) swagger-scala-module project:

https://github.com/swagger-api/swagger-scala-module

https://github.com/swagger-api/swagger-scala-module/blob/develop/src/test/scala/ModelPropertyParserTest.scala#L16
